typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -155,6 +230,7 @@ export abstract class HyperlaneAppGovernor<
     const signerAddress = await signer.getAddress();
 
     const transactionSucceedsFromSender = async (
+      chain: ChainName,
```

Comment:
`chain` is overloaded
I don't think you are passing a different value anywhere

Reply:
origin